examples/accordion/accordion.html

```diff
@@ -273,31 +273,31 @@ <h2 id="rps_label">Role, Property, State, and Tabindex  Attributes</h2>
         <tr>
           <th scope="row"><code>button</code></th>
           <td></td>
-          <td><code>a</code></td>
+          <td><code>button</code></td>
           <td>
-            Identifies the element as a button inside the accordion header that provides hide/show functionality.
+            Button inside the accordion header that provides hide/show functionality.
```

Comment:
Actually, we need to delete this row of the table because we do not apply the role button to the button element. This table lists only explicit uses of ARIA, not implicit roles.

Reply:
Ahh, I actually thought about that. But wasn't sure. Fixed this in commit b35d3a1